src/main/java/com/zaxxer/hikari/pool/PoolEntry.java

```diff
@@ -74,10 +75,21 @@ final class PoolEntry implements IConcurrentBagEntry
    /**
     * Release this entry back to the pool.
     */
-   void recycle()
-   {
+   void recycle() {
+      // endRequest Call just to let driver know.Supported only for JDBC 4.3 and above
```

Comment:
// endRequest Call just to let driver know. (what??)
The comment looks incomplete. Please mention issue# here and complete this sentence.